examples/gno.land/r/sys/params/params.gno

```diff
@@ -0,0 +1,37 @@
+package params
+
+import (
+	"std"
+
+	"gno.land/p/demo/dao"
+	"gno.land/r/gov/dao/bridge"
+)
+
+func NewStringPropExecutor(key string, value string) dao.Executor {
+	return newPropExecutor(key, func() { std.SetParamString(key, value) })
```

Comment:
This is a question mainly for the Params Keeper PR, but how do we guard against malicious Realms or Packages calling `std.SetParamString` on something important, such as the `"gno.land/r/sys/params.sys.users_pkgpath"`?
I can see how changing this value in runtime can make the node state "corrupt", and cause the node to misrepresent its own internal state when proposing blocks or signing

Here is an example:
- user executes some function that called `std.SetParamString` internally and modified this sys users value
- an attacker sends a transaction that deploys a Realm or Package to the now open namespace where they have access
- the node with this corrupted internal param state proposes a block with invalid modifications
- other nodes reject it because it's invalid, and mark the node as malicious

Is this a real scenario?
I need to dig deeper into the Params keeper implementation

Reply:
Realms calling `std.SetParamXXX` can specify an arbitrary key prefixed by the realm's pkgpath. Only `r/sys/params` can set `r/sys/params.XXX`.

Reply:
>Only r/sys/params can set r/sys/params.XXX

Maybe I'm looking at the code wrong, where is this limitation in the Params keeper?

Reply:
https://github.com/moul/gno/blob/dev/moul/r/sys/params/gnovm/stdlibs/std/params.go